docs/demo.md

```diff
@@ -8,8 +8,28 @@
 
 ### App screenshots
 
+
 ![1](https://user-images.githubusercontent.com/33238323/63285214-e60c5780-c2d2-11e9-8111-761bb9265d70.png)
 
+
+
 ![3](https://user-images.githubusercontent.com/33238323/63285291-050ae980-c2d3-11e9-8ec8-3b16c6554299.png)
 
+
+
 ![2](https://user-images.githubusercontent.com/33238323/63285249-f6243700-c2d2-11e9-89fc-0f820a5389b6.png)
+
+
+
+
+
```

Comment:
What's the need for all these new lines?

Reply:
Because gis and screenshots are big enough, just for spacing.